roles/base_os/tasks/main.yaml

```diff
@@ -11,21 +11,26 @@
     src: vimrc
     dest: /root/.vimrc
 
-- name: Ensure vimrc is installed for user root
-  copy:
-    src: vimrc
-    dest: /root/.vimrc
+- name: Add KUBECONFIG to .bash_profile for user root
+  lineinfile:
+    dest: /root/.bash_profile
+    regexp: "KUBECONFIG="
+    line: "export KUBECONFIG={{ openshift_master_credentials_dir }}.kubeconfig"
```

Comment:
In a file that doesn't exist on this branch...  I'm going to push an update to use the explicit path name.